app/controllers/api/players_controller.rb

```diff
@@ -0,0 +1,27 @@
+class Api::PlayersController < ApplicationController
+  skip_before_action :verify_authenticity_token
+  def index 
+    @players = Player.all
```

Comment:
Why are you getting all players?

Reply:
this query, is for profile visor, he can view all players and i add cause is part of CRUD